src/main/webapp/index.html

```diff
@@ -1,105 +1,22 @@
 <!DOCTYPE html>
-<html lang="en">
```

Comment:
why remove this? It's an important accessibility feature

Reply:
ah I think this is a mistake, sorry